app/hooks/useQuestionStateInUrl.ts

```diff
@@ -116,6 +123,7 @@ export default function useQuestionStateInUrl(minLogo: boolean, initialQuestions
 
     const onSiteAnswers = onSiteAnswersRef.current
     const onSiteSet = new Set(onSiteAnswers.map(({pageid}) => pageid))
+    console.debug(relatedQuestions, onSiteSet)
```

Comment:
should this be here?

Reply:
not really, but I no longer care today 🤷‍♀️

feel free to remove in next PR (or I will remove myself when I notice next time)